apypie/inflector.py

```diff
@@ -3,6 +3,11 @@
 
 import re
 
+try:
+    from typing import List, Tuple
```

Comment:
It feels strange to import stuff to use it in comments...

Reply:
It does, but that's how it works.

The `try/except` is there for Python2 as I don't want to introduce a dependency on `typing` (which is external in Py2), and this is not needed at runtime.

Reply:
Also note that when you drop Python 2 you end up with type hints in the actual code and then it does make sense:
```python
def _apply_inflections(self, word: str, rules: List[Tuple[str, str]]) -> str:
```

Reply:
Yeah. *when*… ;)

Reply:
I just thought it'd help to understand it conceptually :)

Reply:
https://www.python.org/dev/peps/pep-0585/ suggests this will be easier in the future